### PR TITLE
"user.home" error

### PR DESCRIPTION
**Pull request by RustamKuramshin**
Fixed "user.home" because "home" key not exist.

### CODE REVIEW

File: tasks/user.yml [hunk 1]
```diff
@@ -82,14 +82,15 @@
 
 - name: create .ssh directory for {{ user.name }}
   file:
-    path: "{{ user.home }}/.ssh"
+    path: "{{ item.home }}/.ssh"
```
Comment (robertdebock) [inline tasks/user.yml:85]: The looped variable is renamed to `user` in `tasks/main.yml`: (See the `loop_var` parameter.)

```yaml
- name: loop over users_user_list
  include: user.yml
  with_items:
    - "{{ users_user_list }}"
  loop_control:
    label: "{{ user.name }}"
    loop_var: user
  when:
    - users_user_list is defined
```

Can you tell me what issue you ran into? I'm pretty sure you are having a legitimate issue with this role.

**Review by robertdebock — COMMENTED** [review]
Thanks for your PR! I don't fully understand what issue you are experiencing.

[CI failed](https://travis-ci.org/robertdebock/ansible-role-users).

File: tasks/user.yml (hunk 1 above)
Comment (RustamKuramshin) [reply tasks/user.yml:85]: Hey. The condition "user.home is defined" is never fulfilled. As a result, the task "create .ssh directory for {{user.name}}" will be skipped.

Comment (robertdebock) [reply tasks/user.yml:85]: Ah, I think I understand what you mean, and indeed you are right. Let me have a look if the role needs a slight refactor for this.

Comment (RustamKuramshin) [reply tasks/user.yml:85]: Use the ansible playbook debugger to verify this (type "p task_vars [" user "] [" home "]" in the debugger console).

**Comment by robertdebock** [comment]
I'm going to close this PR because of inactivity. I think the issue you've spotted has meanwhile been solved differently.

Thanks for you help!

**Pull request closed without merging**